setup/puppeteer_environment.js

```diff
@@ -0,0 +1,40 @@
+// puppeteer_environment.js
+const print = require('./print');
+const NodeEnvironment = require('jest-environment-node');
+const fs = require('fs');
+const path = require('path');
+const puppeteer = require('puppeteer');
+const os = require('os');
+const DIR = path.join(os.tmpdir(), 'jest_puppeteer_global_setup');
+
+class PuppeteerEnvironment extends NodeEnvironment {
+  constructor(config) {
+    super(config);
+  }
+
+  async setup() {
```

Comment:
There are setup function and setup.js file. What is the difference?

Reply:
https://github.com/ilhan-mstf/jest-puppeteer-test-automation/pull/1#discussion_r335477058